appinventor/buildserver/src/com/google/appinventor/buildserver/resources/runtime.scm

```diff
@@ -456,6 +490,17 @@
                         (cdr event-info)))
                      events))
 
+         ;; Register the multi events with the Simple event dispatcher
+         (define (register-multi-events events)
+           (define-alias SimpleEventDispatcher
+             <com.google.appinventor.components.runtime.EventDispatcher>)
+           (for-each (lambda (eventName)
+                       ;; Tell the Simple event dispatcher to delegate dispatching of this event to this class
+                       (SimpleEventDispatcher:registerMultiEventForDelegation
+                        (as com.google.appinventor.components.runtime.HandlesEventDispatching (this))
+                        eventName))
```

Comment:
@gitars

Oops, you're right.  Can you tell me more about what errors, log messages and behavior you're seeing?